esp32_lyrat/esp32_lyrat_idf4.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: CC0-1.0
+ */
+
+#include "esp_err.h"
+#include "bsp/esp32_lyrat.h"
+#include "bsp_err_check.h"
+#include "esp_codec_dev_defaults.h"
+
+static const audio_codec_data_if_t *i2s_data_if = NULL;  /* Codec data interface */
+
+esp_err_t bsp_audio_init(const i2s_config_t *i2s_config)
+{
```

Comment:
Could we check `i2s_data_if != NULL` in the beginning and return ESP_OK if the condition is true?